File: backend/src/controller/controller.js
const pool = require('../config/database');

const listarPontos = async (req, res) => {
    try {
        const [resultado] = await pool.query(
            'SELECT * FROM pontos'
        );

        res.json(resultado);
    } catch (erro) {
        console.error(erro);

        res.status(500).json({
            erro: erro.message
        });
    }
};

module.exports = {
    listarPontos
};

File: backend/src/controller/controller.py
from config.database import get_connection


def testar_banco():
    conexao = None
    cursor = None

    try:
        conexao = get_connection()
        cursor = conexao.cursor(dictionary=True)

        cursor.execute("SELECT 1 AS teste")

        resultado = cursor.fetchall()

        return resultado

    finally:
        if cursor:
            cursor.close()

        if conexao:
            conexao.close()